server/Dockerfile

```diff
@@ -8,6 +8,6 @@ COPY . /home/app
 
 RUN npm install
 
-EXPOSE 8080
+EXPOSE 8080/tcp
```

Comment:
Does /tcp make a difference here?

Reply:
From what I understand, tcp is the default, so no